src/main/resources/config/liquibase/changelog/20230214000000_add_columns_for_new_structure.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <property name="jsonType" value="varchar" dbms="h2"/>
+    <property name="jsonType" value="jsonb" dbms="postgresql"/>
+    <property name="jsonType" value="text" dbms="oracle"/>
+
+    <changeSet id="20230214000000" author="fpyshnyi">
+        <addColumn tableName="xmtimeline">
+            <column name="source" type="varchar(255)"/>
```

Comment:
chould it be 255 bytes or char?

Reply:
I'm sorry, but I don't understand this question. This field will have the same data type as most text fields in the xmtimeline table (PostgreSQL - varchar(255), Oracle - varchar2(255)).

Reply:
for me OK if suitable for your needs.